client/src/pages/cejst.tsx

```diff
@@ -15,45 +15,51 @@ const CEJSTPage = ({location}: IMapPageProps) => {
   //   We will bring back later when we have interactive controls.
   return (
     <Layout location={location}>
-      <main id="main-content" role="main">
-
-        <section>
-          <h2>Just Progress communities</h2>
-          <div className={styles.disclaimer}>
-            <div className={styles.textBox}>
-              <p>
-                Just Progress helps identify and prioritize communities across the United States and U.S. territories
-                that have been historically overburdened and underserved. These communities will receive 40% of
-                the benefits from investments in key areas outlined by the
-                &nbsp;
-                <a
-                  href={'https://www.whitehouse.gov/briefing-room/' +
-                    'presidential-actions/2021/01/27/' +
-                    'executive-order-on-tackling-the-climate-' +
-                    'crisis-at-home-and-abroad/'}
-                  target={'_blank'}
-                  rel={'noreferrer'}>
-                  Executive Order on Tackling the Climate Crisis at Home and Abroad
-                </a>.
-              </p>
-              <p>
-                Download the Just Progress packet or explore the map below to see the list of prioritized communites. To
-                learn more about how these communities were prioritized check out the
-                &nbsp;
-                <a
-                  href={'./methodology'}>
-                  Methodology
-                </a>
-                &nbsp;
-                page.
-              </p>
-            </div>
-            <DownloadPacket />
+      <section>
+        <h2>Just Progress communities</h2>
+        <div className={styles.disclaimer}>
+          <div className={styles.textBox}>
+            <p>
+              Just Progress helps identify and prioritize communities across the
```

Comment:
I know this was here before, but we should make sure this is wrapped with an `intl.formatMessage` - maybe a task ticket for @vim-usds ?

Reply:
Yeah, I started to localize the cejst file, but then thought better of mixing too many things into the "fix header" changelist.

Can I get the PR for the header and then do the cleanup/localization on a different PR?

Reply:
I agree, I think let's make this part of the other ticket on the methodology page